GAE/src/org/waterforpeople/mapping/domain/QuestionAnswerStore.java

```diff
@@ -35,7 +35,7 @@ public class QuestionAnswerStore extends BaseDomain {
     private static final long serialVersionUID = 3726562582080475960L;
 
     @Persistent
-    private Long arbitratyNumber;
+    private Long arbitraryNumber;
```

Comment:
This change may not be possible for the database.

Reply:
This property will be removed. You can mark that with a TODO.